pkg/systemd/logsHelpers.js

```diff
@@ -58,6 +58,13 @@ export const getGrepFiltersFromOptions = ({ options }) => {
             match.push(...['_SYSTEMD_UNIT=' + s, "+", "COREDUMP_UNIT=" + s, "+", "UNIT=" + s]);
         });
         full_grep += "service:" + options.service + " ";
+    } else if (options["user-service"]) {
+        options["user-service"].split(",").forEach(s => {
+            if (!s.endsWith(".service"))
+                s = s + ".service";
+            match.push(...['_SYSTEMD_USER_UNIT=' + s, "+", "COREDUMP_USER_UNIT=" + s, "+", "USER_UNIT=" + s]);
+        });
+        full_grep += "user-service:" + options["user-service"] + " ";
```

Comment:
These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18032-20221216-124213-5d867205-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)